spec/controllers/password_reset_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe PasswordResetsController do
+  describe 'GET show' do
+    it 'renders show template if token is valid' do
+      user = Fabricate(:user)
+      get :show, id: user.token
+      expect(response).to render_template :show
+    end
+
+    it 'redirects to expired token page if the token is not valid' do
+      get :show, id: 'asdfa'
+      expect(response).to redirect_to expired_token_path
+    end
+
+    it 'sets @token' do
+      user = Fabricate(:user)
+      user.update_column(:token, '12345')
+      get :show, id: user.token
+      expect(assigns(:token)).to eq('12345')
+    end
+  end
+
+  describe 'POST create' do
+    context 'with valid token' do
+      it 'update user password' do
+        user = Fabricate(:user)
+        new_password = 'new password'
+        post :create, token: user.token, password: new_password
+        expect(user.reload.authenticate('new password')).to be_present
```

Comment:
since you have set ``new password`` to local variable you could reference that :)